privacyidea/lib/policy.py

```diff
@@ -493,6 +501,13 @@ def get_policies(self, name=None, scope=None, realm=None, active=None,
         :return: list of policies
         :rtype: list of dicts
         """
+        if user_object is not None:
```

Comment:
We could say

    if user_object:

this would also be fales, if user_object would be an empty user_object - right? (see my comment above)

Reply:
I'll try this!